package.json

```diff
@@ -31,6 +31,7 @@
     "lint": "prettier --check . && yarn workspaces -s run lint",
     "build": "yarn workspaces -s run build",
     "test": "yarn workspaces -s run test",
+    "dev:api-docs": "concurrently -n typedoc,http-server 'nodemon -e ts --watch ./packages/connect --watch ./packages/connect-extension-protocol --exec typedoc --entryPoints ./packages/connect/src/index.ts --entryPoints ./packages/connect-extension-protocol/src/index.ts' 'http-server _site/api'",
```

Comment:
Not that this is needed, but why not

Reply:
It's not *needed* but it really helps speed up the documentation workflow.  It's much easier to check the docs look right now by using that script.